tests/functional/spec/features/validation/sum/equal_total_in_separate_section_hub.spec.js

```diff
@@ -0,0 +1,179 @@
+import TotalTurnoverPage from "../../../../generated_pages/validation_sum_against_total_hub_with_dependent_section/total-turnover-block.page";
```

Comment:
I thought we keep these in alphabetical order

Reply:
Not sure if that was formally agreed. I don't mind alphabetical, but I do think logical order is better.